### PR TITLE
updated README.md:

### PR DESCRIPTION
**Pull request by guydebros**
- corrected information about encryption
- made additions based on proposed changes to overview.md
- made other changes for clarity and grammar

See #979, #980, #981

### Checklist

- [x] I have read the [contributing doc](https://github.com/jrnl-org/jrnl/blob/develop/CONTRIBUTING.md).
- [x] I have included a link to the relevant issue number.
- [ ] I have tested this code locally.
- [ ] I have checked to ensure there aren't other open [pull requests](../pulls)
  for the same issue.
- [ ] I have written new tests for these changes, as needed.
- [ ] All tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
...
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+Its goal is to facilitate the rapid creation and viewing of journal entries. It
+is flexible enough to support different use cases and organization strategies.
+It is powerful enough to search through thousands of entries and display, or
+"filter," only the entries you want to see.
 
-### Why keep a journal?
+`jrnl` includes support for [128-bit AES
+encryption](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard) using
+[cryptography.Fernet](https://cryptography.io/en/latest/fernet/).
```
Comment (micahellison) [inline README.md:16]: The 128-bit correction is great, but could you please remove the cryptography.Fernet reference? It's a bit much for the first section of a readme file.

File: README.md [hunk 2]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
...
-professional use, consider a text-based journal to be the perfect complement to
-your GTD todo list - a documentation of what and how you've done it.
+## In a Nutshell
 
-In a Nutshell
--------------
+To make a new entry, just enter
 
-To make a new entry, just type
+``` sh
```
Comment (wren) [inline README.md:22]: Using `sh` here adds syntax highlighting that we don't really want (`in` and `time` are now showing up in a different color.

File: README.md [hunk 3]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
...
-*jrnl* is a simple journal application for your command line. Journals are
-stored as human readable plain text files - you can put them into a Dropbox
-folder for instant syncing and you can be assured that your journal will still
-be readable in 2050, when all your fancy iPad journal applications will long be
-forgotten.
+`jrnl` is a simple journal application for the command line.
 
-Optionally, your journal can be encrypted using the [256-bit
-AES](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
+Its goal is to facilitate the rapid creation and viewing of journal entries. It
```
Comment (micahellison) [inline README.md:9]: Can we use simpler language? We're trying to keep a conversational tone in the documentation, and we especially want the readme to be as easy to read as possible.

File: README.md [hunk 4]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
 _To get help, [submit an issue](https://github.com/jrnl-org/jrnl/issues/new) on
 Github._
 
-*jrnl* is a simple journal application for your command line. Journals are
-stored as human readable plain text files - you can put them into a Dropbox
```
Comment (micahellison) [inline README.md]: I think the fact that jrnl uses plain text files is one if its "killer features"--the reason to use this instead of some other program--and it's worth keeping here.

File: README.md [hunk 5]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
...
 <a href="https://github.com/jrnl-org/jrnl/graphs/contributors"><img
 src="https://opencollective.com/jrnl/contributors.svg?width=890&button=false"
 /></a>
 
-If you'd also like to help make jrnl better, please see our [contributing
+If you'd also like to help make `jrnl` better, please see our [contributing
 documentation](CONTRIBUTING.md).
 
-### Financial Backers
+## Financial Backers
```
Comment (micahellison) [inline README.md:64]: Let's keep this a third-level headline, so "Financial Backers" are part of "Contributors"

File: README.md [hunk 6]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
...
+first sentence ending (either `.`, `?`, or `!`) is interpreted as the title, and
+the rest as the body. In your journal file, the result will look like this:
 
     [2012-03-29 09:00] Called in sick.
-    Used the time cleaning the house and writing my book.
+    Used the time to clean the house and write my book.
 
-If you just call `jrnl`, you will be prompted to compose your entry - but you
-can also configure _jrnl_ to use your external editor.
+Entering `jrnl` without any arguments launches an external editor where you can
```
Comment (micahellison) [inline README.md:34]: I think it's good to keep the documentation more conversational here as well--this reads a bit like formal documentation but we want the readme to be as readable as possible.

File: README.md [hunk 7]
```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
...
-you did them. Your imagination may be limitless, but your memory isn't. For
-personal use, make it a good habit to write at least 20 words a day. Just to
-reflect what made this day special, or why you haven't wasted it. For
-professional use, consider a text-based journal to be the perfect complement to
-your GTD todo list - a documentation of what and how you've done it.
+## In a Nutshell
 
-In a Nutshell
--------------
+To make a new entry, just enter
```
Comment (micahellison) [inline README.md:20]: Replacing "type" with "enter" is a nice accessibility catch!

**Comment by wren** [comment]
Merge conflicts fixed in #1031.
